packages/lesswrong/lib/routes.ts

```diff
@@ -628,6 +628,12 @@ const forumSpecificRoutes = forumSelect<Route[]>({
       getPingback: async (parsedUrl) => await getPostPingbackById(parsedUrl, introPostIdSetting.get()),
       background: postBackground
     },
+    {
+      name: 'bestOf',
+      path: '/bestOf',
+      componentName: 'EABestOfPage',
+      title: 'Best of the Forum',
```

Comment:
I've decided to not gate this in any way (adminOnly or "experimental features"-only) because I thought it might be useful for people to be able to test logged out, or at least without fiddling with their user settings. It's not linked from anywhere so no-one is going to stumble across it, and if they do then that seems fine